liquidctl/driver/smart_device.py

```diff
@@ -627,6 +633,39 @@ def _write_fixed_duty(self, cid, duty):
         self._write(msg)
 
 
+class H1SmartDevice(SmartDevice2):
+    SUPPORTED_DEVICES = [
+        (0x1e71, 0x2015, None, 'NZXT H1 V2', {
+            'speed_channel_count': 2,
+            'color_channel_count': 0
+        }),
+    ]
+
+    def _get_status_directly(self):
```

Comment:
Since there wont be a hwmon driver to support anytime soon, we should probably just override `get_status()`.

Otherwise we have something that appears to support hwmon, but not really, since we have no way of writing the equivalent `_get_status_from_hwmon()` implementation right now.